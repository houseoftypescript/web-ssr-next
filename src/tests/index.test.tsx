import HomePage from '@/pages';
import { render } from '@testing-library/react';

describe('HomePage', () => {
  it('to match snapshot', () => {
    const { container } = render(<HomePage />);
    expect(container).toMatchSnapshot();
  });
});
