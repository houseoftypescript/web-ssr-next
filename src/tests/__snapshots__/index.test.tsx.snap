// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage to match snapshot 1`] = `
<div>
  <main
    class="w-full h-screen flex items-center justify-center"
  >
    <div
      class="flex flex-col gap-4 text-center uppercase"
    >
      <h1
        class="text-9xl font-bold uppercase"
      >
        Next.js
      </h1>
      <p
        class="text-3xl"
      >
        TypeScript + Tailwind CSS + Material UI
      </p>
      <a
        href="/auth/sign-in"
      >
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-fullWidth MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-fullWidth css-1fu7jd5-MuiButtonBase-root-MuiButton-root"
          tabindex="0"
          type="button"
        >
          Sign In
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
      </a>
    </div>
  </main>
</div>
`;
